lib/rubocop/cop/rspec/describe_method.rb

```diff
@@ -3,32 +3,123 @@
 module RuboCop
   module Cop
     module RSpec
-      # Checks that the second argument to `describe` specifies a method.
+      # Checks that arguments and sub-describes of `describe` specify a method.
       #
       # @example
       #   # bad
-      #   describe MyClass, 'do something' do
+      #   describe MyClass do
+      #     describe 'flowers' do; end
+      #   end
+      #
+      #   describe MyClass, 'flowers' do
       #   end
       #
       #   # good
+      #   describe MyClass do
+      #     describe '#my_instance_method' do; end
+      #   end
+      #
+      #   describe MyClass do
+      #     describe '.my_class_method' do; end
+      #   end
+      #
       #   describe MyClass, '#my_instance_method' do
       #   end
       #
       #   describe MyClass, '.my_class_method' do
       #   end
+      #
+      # @example configuration
+      #
+      #   # .rubocop.yml
+      #   RSpec/DescribeMethod:
+      #     IgnoredDescribes:
+      #     - flowers
+      #     - stars
+      #
+      #   # bad
+      #   describe MyClass do
+      #     describe 'not_flowers_or_stars' do; end
+      #   end
+      #
+      #   describe MyClass, 'not_flowers_or_stars' do
+      #   end
+      #
+      #   # good
+      #   describe MyClass do
+      #     describe 'flowers' do; end
+      #     describe 'stars' do; end
+      #     describe '#some_method' do; end
+      #     describe '.some_class_method' do; end
+      #   end
+      #
+      #   describe MyClass, 'flowers' do
+      #   end
+      #
       class DescribeMethod < Cop
         include RuboCop::RSpec::TopLevelDescribe,
                 RuboCop::RSpec::Util
 
-        MESSAGE = 'The second argument to describe should be the method ' \
-                  "being tested. '#instance' or '.class'".freeze
         METHOD_STRING_MATCHER = /^[\#\.].+/
+        MESSAGE = 'The second level describes should test described'\
```

Comment:
How about "nested describes should test.."? I imagine if someone has a third level describe it is also going to flag that?